src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -55,20 +56,31 @@ class WorkspaceSettingsPage extends React.Component {
         if (this.props.policy.isPolicyUpdating) {
             return;
         }
-        const name = values.name.trim();
         const outputCurrency = values.currency;
-        Policy.updateGeneralSettings(this.props.policy.id, name, outputCurrency);
+        Policy.updateGeneralSettings(this.props.policy.id, values.name, outputCurrency);
         Keyboard.dismiss();
     }
 
     validate(values) {
         const errors = {};
-        if (!values.name || !values.name.trim().length) {
+        const name = values.name.trim();
+
+        if (this.hasHtml(name)) {
+            errors.name = 'HTML tags are not allowed in workspace names.';
+        }
+
+        if (!name || !name.length) {
             errors.name = this.props.translate('workspace.editor.nameIsRequiredError');
         }
+
         return errors;
     }
 
+    hasHtml(name) {
```

Comment:
This function is not really necessary given it's a 1 liner called in 1 place

Reply:
Are html tags the only restriction? Do we have any constraints on special characters?

Reply:
Testing the WAF input rules, it seems special characters are cool. We do aggressively go after anything that looks like a tag

Reply:
This function is not really necessary given it's a 1 liner called in 1 place